### DISCUSSION TITLE
Is it possible to turn off all predictions except emoji

### DISCUSSION THREAD

**Discussion by stefanfoulis** [mike-fabian/ibus-typing-booster#292] [Q&A] [answered]
How can I setup the typing booster to give me only emoji suggestions?
I'd like to configure a mode that is dedicated to emoji input. 🚀 😁

**Comment by mike-fabian** [comment, marked as answer]
Yes, that is possible.

Use only dictionaries which do not offer spell checking but only emoji. For example, the “fr” dictionary offers emoji in French, the “fr_FR” dictionary offers emoji **and** spell checking in French. Usually the dictionaries without the region code (here without the “FR”) offer only emoji. If you only care for emoji predictions in English, you may remove **all** dictionaries from the setup, English emoji lookup always works (See also this comment in a closed issue: https://github.com/mike-fabian/ibus-typing-booster/issues/279#issuecomment-1054629927  )

The words you type will still be saved to the user database and be offered as word suggestions. If you don’t want that, you could switch off saving to the user dabase by using the option 

☑️ Off the record mode

and clearing the contents of the user database with the “Delete learned data” button. 

If you use the option

☑️ Unicode symbols and emoji predictions

emoji will always be searched for every word you type. If you also want to type normal words while Typing Booster is active then it is better to switch that option off, then emoji search is only triggered if the typed word starts or ends with an emoji trigger key.  (Default trigger key is underscore, but it is configurable, you can  use other keys like  `:` if you like or use more than one trigger key like `_:`)

If you decide to switch off “☐ Unicode symbols and emoji predictions” and use trigger keys, then you could try to switch off 

☐ Off the record mode

to record which emoji you type often in the user database. Then the order of the emoji offered may change depending on which emoji you choose more often. 

There is a remaining bug with this though, see:

https://github.com/mike-fabian/ibus-typing-booster/issues/289

**Reply by stefanfoulis** [reply]
Thank you for the detailed response. Works great! 🥳
